shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -2246,4 +2249,12 @@ object GlutenConfig {
       .doc("If enabled, gluten will convert the viewfs path to hdfs path in scala side")
       .booleanConf
       .createWithDefault(false)
+
+  val VELOX_BROADCAST_BUILD_RELATION_USE_OFFHEAP =
+    buildConf("spark.gluten.velox.BroadcastBuildRelationUseOffheap.enabled")
```

Comment:
nit: Let's use camel case.

Also could rename with `spark.gluten.velox.offHeapBroadcastBuildRelation`? Note `offHeap` is also camel case to match vanilla Spark's naming convention.